components/cart/CartDrawer.tsx

```
@@ -0,0 +1,231 @@
+  return (
+    <View className="flex-1 justify-center items-center px-8">
+      <View className="w-24 h-24 rounded-full bg-muted/30 justify-center items-center mb-6">
+        <ShoppingCart size={32} color={iconColor} style={{ opacity: 0.5 }} />
+      </View>
+      
+      <Text className="text-xl font-semibold text-foreground mb-2 text-center">
+        Your cart is empty
+      </Text>
+      
+      <Text className="text-muted-foreground text-center mb-8 leading-relaxed">
+        Looks like you haven't added anything to your cart yet
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)